etc/kayobe/environments/ci-multinode/networks.yml

```diff
@@ -81,7 +81,7 @@ internal_vlan: 101
 
 # External network
 external_cidr: 192.168.38.0/24
-external_mtu: 1400
+external_mtu: 1350
```

Comment:
I think @sd109 hit this with his aufn work, and possibly @darmach with his magnum work. It's because we're using VXLAN networks with OVN, but internally it sometimes uses GENEVE tunnels which have a larger header size.

One option is to switch to geneve tenant networks when using OVN, which generally work better.